webapp/src/components/ServerError.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import ErrorPage from './ErrorPage'
+
+export default function ServerError() {
+  return (
+    <ErrorPage>
+      <h2>
+        Uh-oh. <br />
+        We couldn’t retrieve information from the server.
+      </h2>
+      <div className="error-message">
+        <p>Please try again in a few minutes.</p>
+        <br/>
+        <p>
+          If the problem persists, contact us at our&nbsp;
+          <a href="https://chat.decentraland.org" target="_blank" rel="noopener noreferrer">
+            Community Chat
+          </a>
+          &nbsp; or via&nbsp;
+          <a href="https://twitter.com/decentraland" target="_blank" rel="noopener noreferrer">
+            Twitter
+          </a>.
+        </p>
+        <br />
+        <p>
+          Confused about what&#39;s going on? Check out the&nbsp;
+          <a to="https://wiki.decentraland.org/" target="_blank" rel="noopener noreferrer">
```

Comment:
`to` instead of `href`. Still, refer to https://github.com/decentraland/auction/pull/57/commits/aefc4d5da62810bab8de9a0dec9b4de01489cbbc#r156351310